src/v3/src/transformer/i18n/transformAuthenticatorButton.ts

```diff
@@ -28,40 +29,42 @@ export const transformAuthenticatorButton: TransformStepFnWithOptions = ({
 ) => {
   traverseLayout({
     layout: formbag.uischema,
-    predicate: (element) => element.type === 'AuthenticatorButton',
-    callback: (element) => {
+    predicate: (element) => element.type === 'AuthenticatorButtonList',
+    callback: (ele) => {
       const { nextStep } = transaction;
       const { relatesTo, name: stepName } = nextStep!;
       const authenticatorKey = relatesTo?.value?.key;
       const params = getI18NParams(nextStep, authenticatorKey);
 
-      // try get i18nKey without methodType
-      let i18nKey = getI18nKey(`${stepName}.authenticator.${(element as AuthenticatorButtonElement).options.key}`);
-      if (i18nKey) {
-        addTranslation({
-          element, name: 'label', i18nKey, params,
-        });
-        return;
-      }
+      (ele as AuthenticatorButtonListElement).options.buttons.forEach((element) => {
```

Comment:
Can we rename the inner parameter `element` to something else to differentiate it from the `ele` from the callback function? I know you already renamed it to try to avoid the conflict but it is still a bit of a collision. `buttonElement` or similar might work?